chapters/kodi/index.md

```diff
@@ -0,0 +1,269 @@
+# *Architecting Software to Keep the Lazy Ones On the Couch*
+
+*A Report On the Journey of Team Kodi*
+
+<p align="center">
+  <img src="images/Kodi.png" width=700px>
+</p>
+
+*Authors: [Kaj Dreef](github.com/kajdreef), [Menno van der Reek](github.com/mvanderreek), [Koen Schaper](github.com/kpschaper), [Maarten Steinfort](github.com/maartensteinfort)*
```

Comment:
No, it didn't trip over this as far as I can remember, but this was the last change I made. So it could be possible I forgot to check it. It should be resolved now.